docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.6
+
+COPY restic /usr/bin
```

Comment:
This will failed because `restic` is in the root directory of repo while `Dockerfile` is put in `docker`.

The thing is we have another `Dockerfile` already: https://github.com/restic/restic/blob/master/Dockerfile